streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordCollectorImpl.java

```diff
@@ -204,11 +204,21 @@ public void onCompletion(final RecordMetadata metadata,
                 }
             });
         } catch (final TimeoutException e) {
-            log.error("Timeout exception caught when sending record to topic {}. " +
-                "This might happen if the producer cannot send data to the Kafka cluster and thus, " +
-                "its internal buffer fills up. " +
-                "You can increase producer parameter `max.block.ms` to increase this timeout.", topic);
-            throw new StreamsException(String.format("%sFailed to send record to topic %s due to timeout.", logPrefix, topic));
+            log.error(
+                "Timeout exception caught when sending record to topic {}. " +
+                    "This might happen if the producer cannot send data to the Kafka cluster and thus, " +
```

Comment:
nit: why one more indent ?

Reply:
It's just how Idea seems to format multiline expressions... e.g.,

Reply:
Not important -- just look funky to me. We concatenate multiple strings and thus all should have the same indent -- why would we indent the first differently?

Reply:
I remember a while back someone on the internet trying to get everyone to always put operators (and commas) at the beginning of the line for this reason. I think it makes sense. I don't think it caught on in general because it creates syntactic ambiguity in Javascript, but since Java requires semicolons to end a line, it should be fine.

Do you have your IDE set up to create this formatting? Maybe it sounds lazy, but the reason I've formatted it this way is that that's what IDEA does by default. I don't want to spend time curating the number of indent spaces by hand on every code change. I couldn't figure out how to get rid of the extra indent in the multi-line string concatenation. Eg, it even does this:
```java
        final String s =
            "asdf"
                + "qwer"
                + "qwer";
```
which is like the worst outcome.

Reply:
I don't know the IDE setting -- this case is rare enough that I "fix" fit manually if it happens.

Reply:
Let me know if you plan to address or ignore this -- I am fine either way.

Reply:
Ok, I agree with you in principle, but I think I'll just leave it as-is, until I can figure out a way to get the IDE to do it for me.